### ISSUE TITLE
Make language models editor accessible like tree

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#280937]
Make language models editor accessible like tree

- Left / Right arrow on the group should toggle collapse state
- The group should say number of model entries
- Role of the widget should be tree
